crates/lib/Cargo.toml

```
@@ -64,3 +66,28 @@ insta = "1.29.0"
 
 [build-dependencies]
 built = "0.6.1"
+
+[[example]]
+name = "delayed-job-retrieval"
+path = "examples/delayed_job_retrieval.rs"
+
+[[example]]
+name = "execute"
+path = "examples/execute.rs"
+
+[[example]]
+name = "local-compilation-and-execution"
+path = "examples/local.rs"
+
+[[example]]
+name = "parametric-compilation"
+path = "examples/parametric_compilation.rs"
+
+[[example]]
+name = "quil-t"
+path = "examples/quil_t.rs"
+
+[[example]]
+name = "compilation-and-simulation-with-libquil"
+path = "examples/libquil.rs"
+required-features = ["libquil"]
```

I don't think you should need to specify every example like this if you are just specifying the required features for one of them. I think you can get the same effect (with different names) with just

```suggestion

[[example]]
name = "libquil"
required-features = ["libquil"]
```